shell/platform/tizen/flutter_tizen.cc

```diff
@@ -69,10 +67,9 @@ FlutterDesktopEngineRef FlutterDesktopRunEngine(
 }
 
 void FlutterDesktopShutdownEngine(FlutterDesktopEngineRef engine_ref) {
-#ifndef __X64_SHELL__
   flutter::Logger::Stop();
-#endif
-  auto engine = EngineFromHandle(engine_ref);
+
+  auto* engine = EngineFromHandle(engine_ref);
```

Comment:
Q. why did you change this?

Reply:
`EngineFromHandle` returns a pointer of `FlutterTizenEngine`. Although `auto` itself can represent both non-pointer and pointer types, it is always good to be explicit about it.

Reply:
Oh, I think it's a good practice.